src/@chakra-ui/gatsby-plugin/components/Avatar.ts

```diff
@@ -0,0 +1,87 @@
+import {
+  createMultiStyleConfigHelpers,
+  cssVar,
+  defineStyle,
+} from "@chakra-ui/react"
+import { avatarAnatomy } from "@chakra-ui/anatomy"
+import { avatarDefaultTheme, defineMergeStyles } from "./components.utils"
+import { omit, pick } from "lodash"
+
+const { defineMultiStyleConfig, definePartsStyle } =
+  createMultiStyleConfigHelpers(avatarAnatomy.keys)
+
+const { baseStyle: defaultBaseStyle, sizes: defaultSizes } = avatarDefaultTheme
+
+const $border = cssVar("avatar-border-color", "transparent")
+const $mlBySize = cssVar("ml-by-size")
+
+const baseStyleContainer = defineStyle((props) =>
+  defineMergeStyles(defaultBaseStyle?.(props).container, {
+    [$border.variable]: "transparent",
+    borderWidth: "1px",
+    "&:hover, [data-peer]:hover ~ &": {
+      boxShadow: "buttonHover",
+    },
+    _focus: {
+      outline: "4px solid",
+      outlineColor: "primary.hover",
+      outlineOffset: "-1px",
+    },
+    _active: {
+      [$border.variable]: "colors.primary.hover",
+      boxShadow: "none",
+      "& img": {
+        opacity: 0.7,
+      },
+    },
+    "[role='group'] &": {
+      [$border.variable]: "colors.background.base",
+      _notLast: {
+        marginLeft: $mlBySize.reference,
+      },
+    },
+  })
+)
+
+const baseStyleExessLabel = defineStyle((props) =>
+  defineMergeStyles(defaultBaseStyle?.(props).excessLabel, {
+    bg: "body.base",
+    color: "background.base",
+    ms: $mlBySize.reference,
+  })
+)
+
+const baseStyle = definePartsStyle((props) => ({
+  container: baseStyleContainer(props),
+  excessLabel: baseStyleExessLabel(props),
+}))
+
+const USED_SIZES = ["xs", "sm", "md", "lg"] as const
+
+const pickedDefaultSizes: { [k in (typeof USED_SIZES)[number]]?: object } =
+  pick(defaultSizes, ...USED_SIZES)
+
+const sizes = defineMergeStyles(pickedDefaultSizes, {
+  xs: {
+    group: {
+      [$mlBySize.variable]: "space.-1",
+    },
+    excessLabel: {
+      fontSize: "0.563rem",
```

Comment:
Curious about this value. How did you get it?

Reply:
In the DS for the `xs` size avatar, the excess label is showing a font size of `9px` which is this in rem.

This value is not a part of the default fontSize tokens. `2xs` in the default theme is `0.625rem`, and `3xs` is `0.45rem`.